md/en/Options.md

```diff
@@ -128,8 +128,8 @@ cqtdeployer -option1 value1 -option2 list, of, values ​​flag1 flag2 flag3
 | Option                      | Descriptiion                                              |
 |-----------------------------|-----------------------------------------------------------|
 |  -icon [target;val,val]    | Sets path to icon for a targets                           |
-|  -disableRunScript [target;val,val] | Disables a generation of run script for selected targets|
-|  -disableShortCut [target;val,val]  | Disables a generation of shortcut for selected targets |
+|  -disableRunScript [target,target2,target3]  | Disables a generation of run script for selected targets|
+|  -disableShortCut [target,target2,target3]   | Disables a generation of shortcut for selected targets |
```

Comment:
```suggestion
|  -disableRunScript [target,target2,target3]  | Disables the generation of run script for selected targets|
|  -disableShortCut [target,target2,target3]   | Disables the generation of shortcut for selected targets |
```

Users can disable some specific functions, so use "the" instead of "a".

Reply:
i do not sure if this fix is correct, because this proposal is not talking about the function of creating shortcuts, and even the option is a function to disable the function of creating shortcuts.

Reply:
@EndrII Well, I can't understand you because I think what you said is not conflict with my point. So, I'm going to ask a question: why did you choose the article "a"?

Reply:
a - because there is no specificity in this context. 
For example: 
Disables a generation of shortcut for selected targets
and 
Disables the generation of shortcut function for selected targets

Reply:
I spoke with a native speaker, here the correct option would be 'the'.
@pzhlkj6612 Thanks

Reply:
@EndrII OK. I think it would be great if that people could help us rewrite the English doc :D